### PR TITLE
[LM-10971] Added post and comment as variable feature

### PR DESCRIPTION
**Pull request by TyagiLikeMinds**

### CODE REVIEW

File: example/lib/app.dart [hunk 1]
```diff
@@ -55,7 +58,23 @@ class _LMSampleAppState extends State<LMSampleApp> {
                   if (ConnectionState.done == snapshot.connectionState &&
                       snapshot.hasData &&
                       snapshot.data!.success) {
-                    return const LMFeedScreen();
+                    return FutureBuilder(
```
Comment (AnujLM) [inline example/lib/app.dart:61]: we have now three future builders nested in each other, is'nt it a bad practice?
can we do somethind like this :
 ValidateUserResponse? validateUser;
 MemberStateResponse? memberState;
 GetCommunityConfigurationsResponse? communityConfigurations;
// can use a better name here:)
Future<void> setUpUser()async {
validateUser=await LMFeedCore.instance.validateUser();
memberState = await LMFeedCore.instance.getMemberState();
communityConfigurations =await LMFeedCore.instance.getCommunityConfigurations();
}

and then assign this async function to future of FutureBuider

**Review by AnujLM — CHANGES_REQUESTED** [review]

File: example/lib/app.dart (hunk 1 above)
Comment (TyagiLikeMinds) [reply example/lib/app.dart:61]: Sure, there is a function in Future class, Future.wait([]) it takes a list of future argumkents and wait for there response. We just need to check if all the calls are successful or not as they are a blocker for us

Comment (TyagiLikeMinds) [reply example/lib/app.dart:61]: Fixed in the coming PR, we can resolved this conversation for now.

Comment (TyagiLikeMinds) [reply example/lib/app.dart:61]: https://github.com/LikeMindsCommunity/likeminds-feed-flutter/pull/43

File: packages/integration/lib/src/views/feed/feed_screen.dart [hunk 2]
```diff
@@ -75,6 +75,20 @@ class LMFeedScreen extends StatefulWidget {
...
+  String postTitleSmallCap =
+      LMFeedPostUtils.getPostTitle(LMFeedPluralizeWordAction.allSmallSingular);
+
+  String commentTitleFirstCapPlural = LMFeedPostUtils.getCommentTitle(
+      LMFeedPluralizeWordAction.firstLetterCapitalPlural);
+  String commentTitleSmallCapPlural =
+      LMFeedPostUtils.getCommentTitle(LMFeedPluralizeWordAction.allSmallPlural);
+  String commentTitleFirstCapSingular = LMFeedPostUtils.getCommentTitle(
+      LMFeedPluralizeWordAction.firstLetterCapitalSingular);
+  String commentTitleSmallCapSingular = LMFeedPostUtils.getCommentTitle(
```
[the remark this answers is not in the document]
Comment (TyagiLikeMinds) [reply packages/integration/lib/src/views/feed/feed_screen.dart:89]: We are storing the constants in local storage only i.e. the post title and comment title is stored in local storage then based on the action passed we process it

**Comment by divyanshgandhilm** [comment]
Approved!

**Pull request merged**